test/e2e/runner/rpc.go

```diff
@@ -103,14 +114,13 @@ func waitForNode(ctx context.Context, node *e2e.Node, height int64, timeout time
 				curHeight = status.SyncInfo.LatestBlockHeight
 				lastChanged = time.Now()
 			}
-
 			timer.Reset(300 * time.Millisecond)
 		}
 	}
 }
 
-// waitForAllNodes waits for all nodes to become available and catch up to the given block height.
-func waitForAllNodes(ctx context.Context, testnet *e2e.Testnet, height int64, timeout time.Duration) (int64, error) {
+// WaitForAllNodes waits for all nodes to become available and catch up to the given block height.
```

Comment:
Any reason for rendering this method public?

Reply:
This is required by the custom benchmark, when waiting that the system boots up.